ci/drone/linux_script_finalize

```diff
@@ -12,11 +12,25 @@ pip3 install s3cmd
 
 cd build
 
-mv ../LICENSE "$DISTDIR/"
-mv ../zig-cache/langref.html "$DISTDIR/"
+# Produce the experimental std lib documentation.
+mkdir -p "$DISTDIR/docs/std"
+"$DISTDIR/bin/zig" test ../lib/std/std.zig \
+  --zig-lib-dir ../lib \
+  -femit-docs="$DISTDIR/docs/std" \
+  -fno-emit-bin
```

Comment:
Please move this to test_linux_std_Debug so that it runs in parallel to other jobs, avoiding the CI timeout of 2 hours, which is bottlenecked on linux_script_finalize.

Reply:
> Please move this to test_linux_std_Debug so that it runs in parallel to other jobs, avoiding the CI timeout of 2 hours, which is bottlenecked on linux_script_finalize.

Done.